pyproject.toml

```diff
@@ -28,6 +29,12 @@ pylint = "^2.5.3"
 flake8-comprehensions = "^3.2.3"
 pytest-randomly = "^3.4.1"
 pytest-cov = "^2.10.0"
+pytest-factoryboy = "^2.0.3"
+rope = "^0.17.0"
```

Comment:
What is it helpful for?

Reply:
I mean I didn't ask to remove it, just curious :dagger:

Reply:
I didn't know as well, until yesterday I remembered it's a refactoring tool used via coc.vim